packages/edit-site/src/components/dataviews/style.scss

```diff
@@ -8,6 +8,9 @@
 		min-height: 100%;
 	}
 }
+.components-base-control > .components-base-control__field {
```

Comment:
We should scope these styles under `.dataviews-something` so it doesn't leak outside the component. We don't know where it'll be used.

Reply:
I found a way to reset the margin via a component prop https://github.com/WordPress/gutenberg/pull/55992/commits/353ca409f4f5c6d16d9877570bce07c77162b108

Reply:
Excellent! Let's try to avoid using internal components class names as much as possible